IFC4x3/Properties/n/NominalThicknessEnd_2dzrJAQ351fhj1A45yJpqq/Documentation.md

```diff
@@ -0,0 +1 @@
+The nominal thickness of the pavement after a transition from its original value given as Thickness. The size information is provided in addition to the shape representation and the geometric parameters used within. In case of inconsistency between the geometric parameters and the size properties, provided in the attached property set, the geometric parameters take precedence. NominalThicknessEnd location shall be regarded as the pavement object’s placement plus its nominal ength.
```

Comment:
Is 'Thickness' supposed to be 'NominalThickness'?

Reply:
also "ength" at the end should be "length" !

Reply:
Fixed